client/ayon_core/hosts/maya/plugins/publish/extract_model.py

```diff
@@ -70,6 +70,9 @@ def process(self, instance):
                           noIntermediate=True,
                           long=True)
 
+        strip_shader = instance.data.get("strip_shaders", True)
+        self.log.debug("Stripping shaders: %s" % strip_shader)
```

Comment:
Is this debug log too verbose? Does it help debug certain edge cases?

Reply:
I think if you want to debug certain publish output - why your model comes without shaders, having this in the debug log can actually save you some time. But maybe it should be printed out only when strip shaders is enabled.

Reply:
I also wonder whether "strip shaders" is better than reversing the terminology and naming the option "include shaders" or "export shaders" which is how usually export options are shown (whether something is included or not, instead of whether it's stripped?)

Reply:
That term got there from the clients so I guess that is how they think about it. "Exporting models is stripping any shaders on them". What do you think @LiborBatek

Reply:
Totally understand it from the perspective of what `model` family has always been, etc. - But just looking at exporters in general and their terminology usage it's usually "what would you like to include"? That's why I wondered whether a `export materials` flag would be better?

Would love to know what Libor thinks as well. And maybe @dee-ynput from a product perspective.